src/components/common/userFollowSpan.js

```diff
@@ -0,0 +1,76 @@
+import React, { Fragment } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import OverlayTrigger from 'react-bootstrap/OverlayTrigger';
+import Button from 'react-bootstrap/Button';
+import { followOrUnfollow } from '../../redux/actions/userFollowActions';
+
+
+const UnfollowUser = (username, followUnFollowFunc) => {
+  followUnFollowFunc(username, 'Unfollow');
+};
+
+const renderTooltipUnfollow = props => (
+  <div
+    {...props}
+    style={{
+      backgroundColor: 'rgba(0, 0, 0, 0.85)',
+      padding: '10px',
+      color: 'white',
+      borderRadius: 3,
+      ...props.style,
+    }}
```

Comment:
Let's remove this inline styling please. Give the div an id or classname and style it within that.

Reply:
Its how the tooltip was styled in the bootstrap docs. I tried I using a className the  CSS did not take effect.

Reply:
Gotcha.